File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x < 0) {
            return false;
        }

        int orignalNum = x;
        long reversedNum = 0;
        while (x > 0) {
            int lastDigit = x % 10;
            reversedNum = (reversedNum*10) + lastDigit;
            x /= 10;
        }

        return orignalNum == reversedNum;

        return 0;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0: return bool(0)

        orignalNum = x
        newNum = 0

        while (x != 0):
            remainder = x % 10
            newNum = (newNum*10) + remainder
            x /= 10

        if orignalNum == newNum:
            return bool(1)

        return bool(0)